parity-util-mem/tests/derive.rs

```diff
@@ -0,0 +1,31 @@
+#[test]
+#[cfg(feature = "std")]
+fn derive_smoky() {
+	use parity_util_mem::{MallocSizeOf, MallocSizeOfExt};
+
+	#[derive(MallocSizeOf)]
+	struct Trivia {
+		v: Vec<u8>,
+	}
+
+	let t = Trivia { v: vec![0u8; 1024] };
+
+	assert!(t.malloc_size_of() > 1000);
+}
+
```

Comment:
Would be good with a test that exercise the `#[ignore_malloc_size_of = …]` attribute and maybe error messages as well?

Reply:
Added `ignore_malloc_size_of` test